### DISCUSSION TITLE
Is the file created by the build correct?

### DISCUSSION THREAD

**Discussion by OmusubiGX** [libuv/libuv#3613] [Q&A]
I am a very beginner.
I did a debug build of the latest version in VisualStudio 2022 and got the following file.
Is this correct?

uv.dll
uv.exp
uv.lib
uv.pdb
uv_a.lib
uv_a.pdb
uv_run_benchmarks_a.exe
uv_run_benchmarks_a.pdb
uv_run_tests.exe
uv_run_tests.pdb
uv_run_tests_a.exe
uv_run_tests_a.pdb

**Comment by bnoordhuis** [comment]
Looks correct to me.

**Comment by TheCoder68** [comment]
what does it mean uv_a.lib, what is the differance than uv.lib
